### PR TITLE
Use VersionCheck.major() to check 9 and up, and remove TestMisc.isJava9 

### LINKED ISSUE #2073: JDK10 TestManagementUtils.testToStackTraceElementCompositeData TestMonitorInfo.testFrom sanity failures
New failures in the JDK10 sanity tests

```
testToStackTraceElementCompositeData
(from org.openj9.test.java.lang.management.TestManagementUtils took 1 ms)
FAIL

Group(s): level.sanity


[Method Execution Trend Chart]
Exception java.lang.AssertionError

Message: expected:<5> but was:<8>
Stacktrace:


at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.failNotEquals(AssertJUnit.java:364)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:80)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:245)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:252)
at org.openj9.test.java.lang.management.TestManagementUtils.testToStackTraceElementCompositeData(TestManagementUtils.java:240)
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(java.base@10.0.1-internal/Native Method)
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(java.base@10.0.1-internal/NativeMethodAccessorImpl.java:62)
at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(java.base@10.0.1-internal/DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(java.base@10.0.1-internal/Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
```

```
testFrom
(from org.openj9.test.java.lang.management.TestMonitorInfo took 0 ms)
FAIL

Group(s): level.sanity


[Method Execution Trend Chart]
Exception java.lang.IllegalArgumentException

Message: CompositeData object does not have the expected number of attributes
Stacktrace:


at com.ibm.java.lang.management.internal.ManagementUtils.verifyFieldNumber(java.management@10.0.1-internal/ManagementUtils.java:185)
at com.ibm.java.lang.management.internal.StackTraceElementUtil.fromArray(java.management@10.0.1-internal/StackTraceElementUtil.java:176)
at com.ibm.java.lang.management.internal.StackTraceElementUtil.from(java.management@10.0.1-internal/StackTraceElementUtil.java:66)
at java.lang.management.MonitorInfo.from(java.management@10.0.1-internal/MonitorInfo.java:168)
at org.openj9.test.java.lang.management.TestMonitorInfo.testFrom(TestMonitorInfo.java:128)
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(java.base@10.0.1-internal/Native Method)
at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(java.base@10.0.1-internal/NativeMethodAccessorImpl.java:62)
at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(java.base@10.0.1-internal/DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(java.base@10.0.1-internal/Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
```

### PR DESCRIPTION
**Pull request by pshipton**
[ci skip]

Fixes #2073

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8,jdk10

**Comment by pdbain-ibm** [comment]
Should this not go in the test utilities repo per https://github.com/eclipse/openj9/issues/1306?

File: test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestMisc.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 public class TestMisc {
 
 	private static Logger logger = Logger.getLogger(TestMisc.class);
-	public static final boolean isJava9 = "9".equalsIgnoreCase(System.getProperty("java.specification.version"));
+	public static final boolean isJava9AndUp = !"1.8".equals(System.getProperty("java.specification.version"));
```
Comment (DanHeidinga) [inline test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestMisc.java]: Can this be changed to use `VersionCheck.isMajor()`?  https://github.com/eclipse/openj9/blob/f66506e7a528340d176260c3af47c63cf66979ec/test/functional/TestUtilities/src/org/openj9/test/util/VersionCheck.java

That way we only have 1 place that determines the current version

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8,jdk10

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8,jdk10

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8,jdk10

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8,jdk10

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8,jdk10

**Comment by pshipton** [comment]
@llxia @smlambert ready for review

**Pull request merged**